app/database/MetricsDB.scala

```diff
@@ -56,5 +57,58 @@ class MetricsDB(implicit val db: Database) {
       }
 
   def getDistinctNewspaperBooks: Either[ProductionMetricsError, Seq[Option[String]]] =
-    await(db.run(metricsTable.filter(book => !book.newspaperBook.isEmpty && book.newspaperBook =!= "").map(_.newspaperBook).distinct.result))
+    await(db.run(metricsTable.filter(!_.newspaperBook.isEmpty).map(_.newspaperBook).distinct.result))
+
+  def getGroupedWordCounts(implicit filters: MetricsFilters): Either[ProductionMetricsError, List[GroupedWordCount]] = {
```

Comment:
I think documenting in the README how to update word count bands would be a good idea while we're not able to keep them in the config

Reply:
Do you think it's hard to find? It's just a map in the appropriate method and I don't think anyone should change them *too* lightheartedly